attr.js

```diff
@@ -1,5 +1,5 @@
-(function(){
-var forEach = Array.prototype.forEach;
```

Comment:
Removal of this is causing errors further down in `_observe`. Same for in `registry.js`